components/cdc/src/endpoint.rs

```diff
@@ -837,7 +830,7 @@ impl<T: 'static + RaftStoreRouter<E>, E: KvEngine> Endpoint<T, E> {
 
     fn on_min_ts(&mut self, regions: Vec<u64>, min_ts: TimeStamp, current_ts: TimeStamp) {
         // Reset resolved_regions to empty.
-        let resolved_regions = &mut self.resolved_region_heap;
```

Comment:
If we use `self.resolved_region_heap` directly, seems we can avoid `RefCell`.

Reply:
It won't compile, because there is a reference to self (`broadcast_resolved_ts`) after the mut reference.